src/file-5.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:28208/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:28208/project/issues?id=sample&branch=feature6&issues=AWD-NAx2IwxFh3cw-V_C&open=AWD-NAx2IwxFh3cw-V_C)

[ISSUE_KEY]: AWD-NAx2IwxFh3cw-V_C